packages/functions/src/optionSchemas.ts

```diff
@@ -206,15 +226,16 @@ export const optionSchemas: Record<string, CustomFunctionOptionsSchema> = {
         items: {
           type: 'string',
         },
-        minItems: 2,
-        errorMessage: `"xor" and its "properties" option require at least 2-item tuples, i.e. ["id", "name"]`,
+        minItems: 1, // XOR is valid with one item (then it is redundant with 'defined' function)
```

Comment:
```suggestion
        minItems: 2,
        errorMessage: `"xor" and its "properties" option require at least 2-item tuples, i.e. ["id", "name"]`,
```
IMO, XOR is requiring at least 2 items to compare. We'd be recommending the use of **defined** for singular use case